examples/pwm.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "wiringX.h"
+
+int main(void) {
+	int pin = 4;
+
+	wiringXSetup();
+	pinMode(pin, PWM_OUTPUT);
+
+	wiringXPWMEnable(pin, 0);
+	wiringXSetPWM(pin, 1000, 50);
+	wiringXPWMEnable(pin, 1);
+
+	sleep(15);
```

Comment:
To output PWM for 15 sec and then disable it. If it's not disabled, it will output PWM all the time even though the program is ceased.